go.mod

```diff
@@ -4,16 +4,14 @@ go 1.22.0
 
 require (
 	github.com/RichardKnop/machinery/v2 v2.0.13
-	github.com/brianvoe/gofakeit/v7 v7.1.1
-	github.com/brianvoe/gofakeit/v7 v7.1.1
+	github.com/brianvoe/gofakeit/v6 v6.28.0
```

Comment:
Downgrade?

Reply:
Must because of my local clone with upgrade it